code/modules/clothing/masks/hailer.dm

```diff
@@ -39,28 +41,161 @@
 	if(istype(W, /obj/item/weapon/screwdriver))
 		switch(aggressiveness)
 			if(1)
-				user << "<span class='notice'>You set the restrictor to the middle position.</span>"
+				user << "<span class='notice'>You set the aggressiveness restrictor to the second position.</span>"
 				aggressiveness = 2
 			if(2)
-				user << "<span class='notice'>You set the restrictor to the last position.</span>"
+				user << "<span class='notice'>You set the aggressiveness restrictor to the third position.</span>"
 				aggressiveness = 3
 			if(3)
-				user << "<span class='notice'>You set the restrictor to the first position.</span>"
-				aggressiveness = 1
+				user << "<span class='notice'>You set the aggressiveness restrictor to the fourth position.</span>"
+				aggressiveness = 4
 			if(4)
+				user << "<span class='notice'>You set the aggressiveness restrictor to the first position.</span>"
+				aggressiveness = 1
+			if(5)
 				user << "<span class='danger'>You adjust the restrictor but nothing happens, probably because it's broken.</span>"
 	else if(istype(W, /obj/item/weapon/wirecutters))
-		if(aggressiveness != 4)
+		if(aggressiveness != 5)
 			user << "<span class='danger'>You broke the restrictor!</span>"
-			aggressiveness = 4
+			aggressiveness = 5
 	else
 		..()
 
 /obj/item/clothing/mask/gas/sechailer/ui_action_click(mob/user, actiontype)
 	if(actiontype == /datum/action/item_action/halt)
 		halt()
-	else
+	else if(actiontype == /datum/action/item_action/adjust)
 		adjustmask(user)
+	else if(actiontype == /datum/action/item_action/selectphrase)
+		switch(aggressiveness)
+			if(1)
+				switch(phrase)
+					if(1)
+						user << "<span class='notice'>You set the restrictor to: Stop in the name of the Law.</span>"
```

Comment:
@uraniummeltdown is there anything you can do to make this code less long and nested